static/app/views/replays/detail/network/index.tsx

```diff
@@ -246,16 +313,64 @@ function NetworkList({replayRecord, networkSpans}: Props) {
           query={searchTerm}
         />
       </NetworkFilters>
-      <StyledPanelTable
-        columns={columns.length}
-        isEmpty={networkData.length === 0}
-        emptyMessage={t('No related network requests found.')}
-        headers={columns}
-        disablePadding
-        stickyHeaders
-      >
-        {networkData.map(renderTableRow)}
-      </StyledPanelTable>
+
+      <NetworkTable ref={networkTableRef}>
+        <AutoSizer>
+          {({width, height}) => (
+            <MultiGrid
+              ref={multiGridRef}
+              columnCount={columns.length}
+              columnWidth={({index}) => {
+                if (index === 0) {
+                  // For the status code column, we want to set a fixed width
+                  return Math.max(80, cache.columnWidth({index}));
```

Comment:
Circling back on this `80` thing.. it looks like it's required so that we don't need to re-calc the width of everything when sorting changes.

We could preserve the space in the header with invisible elements. Something like is what i'm thinking and would play with:
```diff
  const sortArrow = (sortedBy: string) => {
    return sortConfig.by === sortedBy ? (
      <IconArrow
        color="gray300"
        size="xs"
        direction={sortConfig.by === sortedBy && !sortConfig.asc ? 'down' : 'up'}
      />
-   ) : null;
+   ) : (
+     <IconArrow size="xs" style={{visibility: 'hidden'}} />
+   );
  };
```

Reply:
This is indeed way better! I also added a Math.max to the logic for the calc of the width for the second column in order to have a min width, so we can avoid this case scenario:

![image](https://user-images.githubusercontent.com/39612839/197820268-17acae7d-7aea-4b3e-8a27-0a23d41504e4.png)

And instead have: 

![image](https://user-images.githubusercontent.com/39612839/197820476-37ed9d94-3ba0-463d-bd92-780c50e1d744.png)